ansible/roles/kong-api/defaults/main.yml

```diff
@@ -8005,3 +8007,57 @@ kong_apis:
         config.limit_by: credential
       - name: request-size-limiting
         config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: createRCCertificate
+    uris: "{{ registry_service_prefix }}/certificate/v1/create"
+    upstream_url: "{{ vm_registry_service_url }}/api/v1/TrainingCertificate"
+    strip_uri: true
+    plugins:
+      - name: jwt
+      - name: cors
+      - "{{ statsd_pulgin }}"
+      - name: acl
+        config.whitelist:
+          - 'createRCCertificate'
+      - name: rate-limiting
+        config.policy: local
+        config.hour: "{{ medium_rate_limit_per_hour }}"
+        config.limit_by: credential
+      - name: request-size-limiting
+        config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: searchRCCertificate
+    uris: "{{ registry_service_prefix }}/certificate/v1/search"
+    upstream_url: "{{ vm_registry_service_url }}/api/v1/TrainingCertificate/search"
+    strip_uri: true
+    plugins:
+      - name: jwt
+      - name: cors
+      - "{{ statsd_pulgin }}"
+      - name: acl
+        config.whitelist:
+          - 'searchRCCertificate'
```

Comment:
@amit-tarento please change the acl name to certificateAccess same as given to searchRegCertificate API